CHANGELOG.md

```diff
@@ -31,6 +39,318 @@ As of this change, in conformance with the JSON-RPC 2.0 specification it will re
+- chore: bump build version in master (#11946) ([filecoin-project/lotus#11946](https://github.com/filecoin-project/lotus/pull/11946))
+- fix: test: no snap deals in immutable deadlines (#12071) ([filecoin-project/lotus#12071](https://github.com/filecoin-project/lotus/pull/12071))
+- test: actors: manual CC onboarding and proving integration test (#12017) ([filecoin-project/lotus#12017](https://github.com/filecoin-project/lotus/pull/12017))
+- fix: ci: keep lotus checkout clean in the release workflow (#12028) ([filecoin-project/lotus#12028](https://github.com/filecoin-project/lotus/pull/12028))
+- feat!: build: separate miner and node version strings
+- chore: lint: address feedback from reviews
+- chore: lint: fix lint errors with new linting config
+- chore: lint: update golangci lint config
+- ci: fix when sorted pr checks workflow is executed
+- doc: eth: restore comment lost in linter cleanup
+- fix: ci: publish correct docker tags on workflow dispatch (#12060) ([filecoin-project/lotus#12060](https://github.com/filecoin-project/lotus/pull/12060))
+- feat: libp2p: Lotus stream cleanup (#11993) ([filecoin-project/lotus#11993](https://github.com/filecoin-project/lotus/pull/11993))
+- Update SupportedProofTypes (#11988) ([filecoin-project/lotus#11988](https://github.com/filecoin-project/lotus/pull/11988))
+- Revert "Update SupportedProofTypes (#11988)" (#11990) ([filecoin-project/lotus#11990](https://github.com/filecoin-project/lotus/pull/11990))
+- chore: docs: Update skeleton guide (#11960) ([filecoin-project/lotus#11960](https://github.com/filecoin-project/lotus/pull/11960))
+- chore: ci: request contents read permissions explicitly in gha (#12055) ([filecoin-project/lotus#12055](https://github.com/filecoin-project/lotus/pull/12055))
+- fix: ci: use custom GITHUB_TOKEN for GoReleaser (#12059) ([filecoin-project/lotus#12059](https://github.com/filecoin-project/lotus/pull/12059))
+- chore: pin golanglint-ci to v1.58.2 (#12054) ([filecoin-project/lotus#12054](https://github.com/filecoin-project/lotus/pull/12054))
+- chore: fix some function names (#12031) ([filecoin-project/lotus#12031](https://github.com/filecoin-project/lotus/pull/12031))
+- src: lint: bump golangci-lint to 1.59, address unchecked fmt.Fprint*
+
+## Contributors
+
+| Contributor | Commits | Lines ± | Files Changed |
+|-------------|---------|---------|---------------|
+| Aarsh Shah | 9 | +5710/-35899 | 201 |
+| Łukasz Magiera | 21 | +1891/-33776 | 335 |
+| LexLuthr | 9 | +4916/-1637 | 107 |
+| Phi-rjan | 9 | +3544/-187 | 92 |
+| Rod Vagg | 15 | +2183/-479 | 164 |
+| Piotr Galar | 6 | +130/-2386 | 30 |
+| Andrew Jackson (Ajax) | 6 | +1072/-533 | 63 |
+| ZenGround0 | 1 | +235/-13 | 3 |
+| Hubert Bugaj | 3 | +57/-37 | 5 |
+| Steven Allen | 3 | +25/-15 | 6 |
+| Peter Rabbitson | 1 | +16/-8 | 4 |
+| tomfees | 1 | +6/-6 | 5 |
+| imxyb | 1 | +6/-0 | 1 |
+| yumeiyin | 1 | +2/-2 | 2 |
+| galargh | 1 | +2/-2 | 1 |
+
+# v1.27.0 / 2024-05-27
```

Comment:
what's the branching strategy that's brought in the 1.27.0 changes if they weren't already in here? this seems odd, are we branching off the wrong thing, or did we not merge 1.27.0 changes back?

Reply:
The `release/v1.27.1` branch was created off master on Friday, but the https://github.com/filecoin-project/lotus/pull/12076 did not land until yesterday.

I was planning to branch the `release/v1.27.1` off master again after that changes from v1.27.0 landed in master, but it is a protected branch, so I could not do delete and start it off again.